crl-janitor/sa.tf

```diff
@@ -25,6 +25,8 @@ locals {
     "roles/monitoring.editor",
     # Allow tracing.
     "roles/cloudtrace.agent",
+    # Allow deleting project.
+    "roles/resourcemanager.projectDeleter",
```

Comment:
projectDeleter needs to be on a folder or org. It doesn't make sense for it to be on a project, because what projects does that allow it to be delete.

Let's start a list of folder roles like we have `locals.app_sa_roles` to use to create `resource "google_folder_iam_member" "app_folder_roles"` . We already have roles/editor on line59.

Reply:
I also realized that. Thanks! Done